terminator/Tiltfile

```diff
@@ -14,15 +13,14 @@ docker_build("glitch", ".", dockerfile="glitch.dockerfile",
     run('go build glitch/main.go'),
     restart_container(),
   ])
-docker_build("rectangler", ".", dockerfile="rectangler.dockerfile", only=['rectangler', 'render/api'] + shared_deps)
-docker_build("red", ".", dockerfile="red.dockerfile")
+docker_build("rectangler", ".", dockerfile="rectangler.dockerfile", only=['rectangler', 'render/api', 'render/Inconsolata-Bold.ttf'] + shared_deps)
```

Comment:
I might move the ttf out to like /assets/fonts - it'd be annoying to have to update the tiltfile because we changed the font name

or actually can we just put it in rectangler? is it now the only thing that uses it?